data/json/npcs/Lighthouse_Family/NPC_lighthouse_man.json

```diff
@@ -0,0 +1,464 @@
+      [ "fish_trap", 4 ],
+      [ "fishing_hook_bone", 2 ],
+      [ "fishing_rod_basic", 1 ]
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": [
+      "TALK_lighthouse_man_1",
+      "TALK_lighthouse_man_Background",
+      "TALK_lighthouse_man_hay",
+      "TALK_lighthouse_man_survivors_yes",
+      "TALK_lighthouse_man_survivors_no"
+    ],
+    "dynamic_line": "<greet>",
+    "responses": [
+      { "text": "About that job…", "condition": "has_assigned_mission", "topic": "TALK_MISSION_INQUIRE" },
+      { "text": "<BGSS_intro_question>", "topic": "TALK_lighthouse_man_Background" },
+      {
+        "text": "How are you doing?",
+        "topic": "TALK_lighthouse_man_hay",
+        "condition": { "npc_has_var": "helped_lighthouse_man", "type": "flag", "context": "help", "value": "yes" }
+      },
+      { "text": "Do you need any help?", "topic": "TALK_MISSION_LIST" },
+      {
+        "text": "Care to trade?",
+        "topic": "TALK_lighthouse_man_1",
+        "condition": { "npc_has_var": "helped_lighthouse_man", "type": "flag", "context": "help", "value": "yes" },
+        "effect": "start_trade"
+      },
+      {
+        "text": "Has anyone come recently?",
+        "condition": { "u_has_var": "lighthouse_fixed", "type": "global", "context": "safe_space", "value": "yes" },
+        "trial": {
+          "type": "CONDITION",
+          "condition": {
+            "not": { "npc_compare_time_since_var": "wait", "type": "time", "context": "survivors", "op": "<", "time": "14 days" }
+          }
+        },
+        "success": {
+          "topic": "TALK_lighthouse_man_survivors_yes",
+          "effect": [
+            { "npc_add_var": "wait", "type": "time", "context": "survivors", "time": true },
+            { "mapgen_update": "lighthouse_man_survivor", "om_terrain": "lighthouse_ground" }
+          ]
+        },
+        "failure": { "topic": "TALK_lighthouse_man_survivors_no" }
+      },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_lighthouse_man_Background",
+    "dynamic_line": "I'm just a simple man from a simple family.  Before <the_cataclysm> I had a fishing boat, and I went to sea on it, to catch fish and other marine life.  This is how my grandfather lived, my father lived, and I began to live like that.  I had a large family, 7 children, but not many survived, only my wife and daughter remained…"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_lighthouse_man_hay",
+    "dynamic_line": "You know, <very> good.  It is cozy here, there is a lot of space, and most importantly, its safe.  We even started considering this place as our new home.  By the way, while I was fishing, a bottle with a piece of paper came to the shore of our island.  Here, take a look.\", he gave you a piece of paper, something was written on it: '0030045-0030045-35321700', with something similar to lightning scribbled nearby.  \"I wonder what it could mean…"
```

Comment:
```suggestion
    "dynamic_line": "You know, <very> good.  It is cozy here, there is a lot of space, and most importantly, it's safe.  We even started considering this place as our new home.  By the way, while I was fishing, a bottle with a piece of paper came to the shore of our island.  Here, take a look.\", he gave you a piece of paper, something was written on it: '0030045-0030045-35321700', with something similar to lightning scribbled nearby.  \"I wonder what it could mean…"
```